packages/main/src/plugin/extension/extension-loader.spec.ts

```
@@ -35,48 +35,48 @@ import type { ApiSenderType } from '@podman-desktop/core-api/api-sender';
+import type { ImageCheckerImpl } from '/@/plugin/image-checker.js';
+import type { ImageFilesRegistry } from '/@/plugin/image-files-registry.js';
+import type { ImageRegistry } from '/@/plugin/image-registry.js';
+import type { InputQuickPickRegistry } from '/@/plugin/input-quickpick/input-quickpick-registry.js';
 import type { KubeGeneratorRegistry } from '/@/plugin/kubernetes/kube-generator-registry.js';
+import type { KubernetesClient } from '/@/plugin/kubernetes/kubernetes-client.js';
+import type { MenuRegistry } from '/@/plugin/menu-registry.js';
+import type { MessageBox } from '/@/plugin/message-box.js';
 import { NavigationManager } from '/@/plugin/navigation/navigation-manager.js';
+import type { OnboardingRegistry } from '/@/plugin/onboarding-registry.js';
+import type { ProviderRegistry } from '/@/plugin/provider-registry.js';
+import type { Proxy } from '/@/plugin/proxy.js';
```

Biome (2.3.14)

[error] Do not shadow the global "Proxy" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)